client/src/themes/theme.js

```diff
@@ -4,9 +4,31 @@ export const theme = createMuiTheme({
   typography: {
     fontFamily: ["Montserrat", "sans-serif"].join(","),
     fontSize: 12,
-    h1: {
-      // could customize the h1 variant as well
+    h1: {},
+    h2: {},
+    h3: {},
+    h4: {
+      fontWeight: "medium",
     },
+    h5: {
+      fontWeight: "medium",
+    },
+    h6: {},
+    body1: {
+      color: "#000",
+      fontWeight: "bold",
+      fontSize: "14px",
+      lineHeight: "1.6",
+    },
+    body2: {
+      color: "#000",
+      opacity: "0.403921568627451",
```

Comment:
What is the purpose of this line? It's causing the login & signup page to be faded.
<img width="1418" alt="Screen Shot 2021-03-24 at 8 17 17 AM" src="https://user-images.githubusercontent.com/48573314/112309128-5a5e6800-8c79-11eb-9d20-aa2f0e0f914b.png">

Reply:
![Screen Shot 2021-03-24 at 8 32 38 AM](https://user-images.githubusercontent.com/35951681/112310953-8844ac00-8c7b-11eb-9432-c6e6450af064.png)
 
I copied it based on the .sketch mockup. 

![Screen Shot 2021-03-24 at 8 34 54 AM](https://user-images.githubusercontent.com/35951681/112312041-bb3b6f80-8c7c-11eb-9462-36adb2d4fb0e.png)


I am not sure why its making the login and signup page to fade since there is no Typography with variant body2 used there but  I fixed it by removing the <CssBaseline/> inside the Login page and SIgnUp page. I don't think we need the default css from <CssBaseline/> since we are theming and customizing styles each Component.